app/angular/src/demo/welcome.component.ts

```diff
@@ -46,34 +46,34 @@ import { Component, Output, EventEmitter } from '@angular/core';
   `,
   styles: [
     `
-        main {
-          margin: 15px;
-          max-width: 600;
-          lineHeight: 1.4;
-          fontFamily: "Helvetica Neue", Helvetica, "Segoe UI", Arial, freesans, sans-serif;
-        }
+      main {
+        margin: 15px;
+        max-width: 600;
+        lineheight: 1.4;
```

Comment:
I'm guessing this is a prettier change? Is this a bug? (shouldn't it be `line-height`?)

Reply:
indeed it should

Reply:
I guess the problem was there before ;)

Reply:
✅